integration/tests/basic/helpers/rpc_checks.py

```diff
@@ -92,4 +92,7 @@ def assert_log_field_in_neon_trx_receipt(response, events_count):
 
 def assert_fields_are_hex(object, expected_hex_fields):
     for field in expected_hex_fields:
-        assert is_hex(object[field]), f"field {field} is not correct. Actual : {object[field]}"
+        try:
```

Comment:
Don't use try/ecept if you can rewrite this code on if/else. More useful like:
```
if field in object and not is_hex(object[field]): raise
if hasattr(field, object) ...
````

Reply:
Fixed